packages/market-service/src/fiat-market-service.test.ts

```diff
@@ -0,0 +1,49 @@
+import { HistoryTimeframe } from '@shapeshiftoss/types'
+
+import { mockERHFindByFiatSymbol, mockERHPriceHistoryData } from './exchange-rates-host/erhMockData'
+import { FiatMarketProviders } from './fiat-market-providers'
+import { findByFiatSymbol, findPriceHistoryByFiatSymbol } from './fiat-market-service-manager'
+
+jest.mock('./exchange-rates-host/exchange-rates-host', () => ({
+  ExchangeRateHostService: jest.fn().mockImplementation(() => {
+    return {
+      findByFiatSymbol: jest.fn(() => mockERHFindByFiatSymbol),
+      findPriceHistoryByFiatSymbol: jest.fn(() => mockERHPriceHistoryData)
+    }
+  })
+}))
+
+describe('fiat market service', () => {
+  describe('findByFiatSymbol', () => {
```

Comment:
I'd like to see more tests, especially covering error conditions.

Reply:
added a test case for handling `NaN` price from history API. useless validate date check removed.